code-challenges.test.js

```diff
@@ -69,5 +108,23 @@ const accountTransactions3 = []
 // Expected output: []
 
 // b) Create the function that makes the test pass.
+const sumIt = (input) => {
+  let arr = []
+  arr.push(input[0])
+  for(let i = 1; i < input.length ; i++) {
+    console.log(arr);
```

Comment:
Be sure to remove console.logs when you're done using them. Nice work separating the accumulated sum from the input array